common/lib/transport/comettransport.js

```diff
@@ -92,10 +92,10 @@ var CometTransport = (function() {
 		this.dispose();
 	};
 
-	CometTransport.prototype.abort = function() {
-		Logger.logAction(Logger.LOG_MINOR, 'CometTransport.abort()', '');
+	CometTransport.prototype.abort = function(err) {
+		Logger.logAction(Logger.LOG_MINOR, 'CometTransport.abort()', 'err: ' + JSON.stringify(err));
```

Comment:
We have `Utils.inspect()` for this kind of thing. Or, if we know that `err` is certain to be an `ErrorInfo`, then just `err` will do because `toString()` does what we want.